src/components/cardPreview/CardPreview.jsx

```diff
@@ -17,7 +17,6 @@ class Card extends Component {
         <EditField
           text={this.props.title}
           save={(newTitle) => { this.props.saveCardTitle(this.props.id, newTitle) }}
-          dragHandleProps={this.props.dragHandleProps}
```

Comment:
You remove this prop, but in `EditField` it's always required.

Reply:
That's useless : you can still drag&drop cards by the header without this props.

Reply:
Nevermind ! I noticed the error in JS console. Fixed in commit e8dd1c8